docs/content/en/getting_started/upgrading.md

```diff
@@ -61,6 +61,10 @@ godojo installations
 
 If you have installed DefectDojo on "iron" and wish to upgrade the installation, please see the [instructions in the repo](https://github.com/DefectDojo/godojo/blob/master/docs-and-scripts/upgrading.md).
 
+## Upgrading to DefectDojo Version 2.9.x.
+
+**Breaking change for APIv2:** From API endpoint `/api/v2/tool_configurations/` was removed `configuration_url` because of redundancy.
```

Comment:
This would sound a bit better:

**Breaking change for APIv2:** `configuration_url` was removed from API endpoint `/api/v2/tool_configurations/` due to redundancy.